scripts/run_examples.py

```diff
@@ -81,16 +96,23 @@ def run_examples():
             elif ext == 'exe':
                 cmd = ['cmd.exe', '/c', 'start', '/wait', fpath, '/S', '/D=%s' % env_dir]
             errored += _execute(cmd)
+            if keep_artifacts:
+                shutil.move(fpath, keep_artifacts)
         print('')
 
     if errored:
         print('Some examples failed!')
-        print('Assets saved in: %s' % output_dir)
-        sys.exit(1)
+        print('Assets saved in: %s' % parent_output)
     else:
         print('All examples ran successfully!')
-        rm_rf(output_dir)
+        shutil.rmtree(parent_output)
+    return errored
 
 
 if __name__ == '__main__':
-    run_examples()
+    if sys.argv[1].startswith('--keep-artifacts='):
```

Comment:
This should check that `len(sys.argv) >=2` before accessing the element.

Reply:
Good catch!